docs/xdist.rst

```diff
@@ -5,9 +5,9 @@ Distributed testing (xdist)
 "load" mode
 ===========
 
-Distributed testing with dist mode set to load will report on the combined coverage of all slaves.
-The slaves may be spread out over any number of hosts and each slave may be located anywhere on the
-file system.  Each slave will have its subprocesses measured.
+Distributed testing with dist mode set to load will report on the combined coverage of all workers.
```

Comment:
```suggestion
Distributed testing with dist mode set to "load" will report on the combined coverage of all workers.
```

Reply:
I think this change is out of scope

Reply:
?
It is ok to fix typos / documentation when touching it anyway..

You / your PR made me look at that and I've seen it, so it should be included I think - there's certainly not a reason to create a new PR for that (especially given the number of jobs it would run ;))